lib/ssh_scan/api.rb

```diff
@@ -140,7 +147,7 @@ class API < Sinatra::Base
         uuid = params['uuid']
 
         if worker_id.empty? || uuid.empty?
-          return {"accepted" => "false"}.to_json
+          return {"completed" => "false"}.to_json
         end
 
         settings.db.find_work_result(worker_id, uuid)
```

Comment:
Is this even necessary? I cannot think of why the API would need to retrieve a scan result based on worker_id and uuid.  Shouldn't uuid be enough?